decidim-meetings/app/views/devise/mailer/join_meeting.text.erb

```diff
@@ -2,6 +2,10 @@
 
 <%= t("decidim.meetings.admin.invite_join_meeting_mailer.invite.invited_you_to_join_a_meeting", invited_by: @resource.invited_by.name, application: @resource.organization.name) %>
 
+<%= t("devise.mailer.invitation_instructions.decline") %>:
+<% accept_invitation_url(@resource, invitation_token: @token, invite_redirect: Decidim::EngineRouter.main_proxy(@opts[:meeting].component).decline_invitation_meeting_registration_path(meeting_id: @opts[:meeting], participatory_space_id: @opts[:meeting].component.participatory_space), host: @resource.organization.host) %>
```

Comment:
Same

Reply:
I think that we can't access to `decicim_mettings` because this mailers is executed inside `devise`.